server/src/main/java/org/dst/server/service/DstSetServiceImpl.java

```diff
@@ -35,4 +35,51 @@ public DstSetServiceImpl(KVStore kvStore) {
     setGetResponseBuilder.setStatus("ok");
     return setGetResponseBuilder.build();
   }
+
+  @Override
+  public SetProtocol.SetDeleteResponse delete(SetProtocol.SetDeleteRequest request) {
+    SetProtocol.SetDeleteResponse.Builder setDeleteResponseBuilder =
+            SetProtocol.SetDeleteResponse.newBuilder();
+
+    if (store.set().del(request.getKey(), request.getEntity())) {
```

Comment:
Will `del` throw exception?

Reply:
No. If key is not found in db, this method will return `Status.KEY_NOT_FOUND` now.

Reply:
But you use as a boolean here.